LongWarOfTheChosen/Src/LW_Overhaul/Classes/UIScreenListener_ViewObjectives.uc

```diff
@@ -1,54 +1,115 @@
 //---------------------------------------------------------------------------------------
 //  FILE:    UIScreenListener_ViewObjectives.uc
-//  AUTHOR:  Amineri (Pavonis Interactive)
-//  PURPOSE: Adjust objectives pane to use scrollbar instead of slow autoscroll
+//  AUTHOR:  KDM
+//  PURPOSE: The objective screen modified such that objective descriptions are scrollable
+//			 for mouse and keyboard users.
 //---------------------------------------------------------------------------------------
 
 class UIScreenListener_ViewObjectives extends UIScreenListener config(LW_Overhaul);
 
+var UIViewObjectives ObjectiveScreen;
+var UIPanel MouseEventPanel;
+
 event OnInit(UIScreen Screen)
 {
-	MakeDescriptionUseScrollbar(UIViewObjectives(Screen));
+	// KDM : Controller users keep the auto-scroll behaviour built into the objective screen.
```

Comment:
Isn't this kind of frustrating? I've noticed other games linking the scroll to a stick or the D-pad. Is that feasible?

Reply:
I'll try to take a look in the next few days and see how easy it is to do.

Reply:
Done. I did something similar in my Squad Management mod, so it was easier than I thought.